doc/user/content/sql/create-sink.md

```diff
@@ -42,9 +42,11 @@ Field | Use
 ------|-----
 **KAFKA BROKER** _host_ | The Kafka broker's host name without the security protocol, which is specified by the [`WITH` options](#with-options).) If you wish to specify multiple brokers (bootstrap servers) as an additional safeguard, use a comma-separated list. For example: `localhost:9092, localhost:9093`.
 **TOPIC** _topic&lowbar;prefix_ | The prefix used to generate the Kafka topic name to create and write to.
+**KEY (** _key&lowbar;column&lowbar;list_ **)** | An optional list of columns to use for the Kafka key. If unspecified, the Kafka key is left unset. {{< version-added v0.5.1 />}}
 **WITH OPTIONS (** _option&lowbar;_ **)** | Options affecting sink creation. For more details see [`WITH` options](#with-options).
+**CONSISTENCY TOPIC** _consistency&lowbar;topic_ | The Kafka topic name to write [consistency metadata](#consistency-metadata) to.
```

Comment:
I think we should also forward the description from `consistency_topic`:

```
Makes the sink emit additional consistency metadata. Only valid for Kafka sinks. If reuse_topic is true, a default consistency_topic will be used when not explicitly set. The default consistency topic name is formed by appending -consistency to the output topic name.
```

And further below there is a section that mentions "Instead, the topic name specified via the consistency_topic option is used as is.". We should update that to be agnostic of how the consistency topic is specified.

Reply:
> We should update that to be agnostic of how the consistency topic is specified.

Totally! Right now I'm not doing anything with the `CONSISTENCY TOPIC` information. I'll put that docs change in the next PR when I actually use it, if that's okay with you!

Reply:
But I made the first change!

Reply:
Yes, that sounds good!